module1.js

```diff
@@ -110,7 +124,7 @@ function question7(array1, array2, extraElement) {
 */
 
 function question8(retailPrice, taxAmount) {
-  // TODO:
+  return '£' + retailPrice + taxAmount;
```

Comment:
How would you calculate a price of something with added tax? Say you want to buy £20 tshirt online, but you need to add 5% (0.05) tax on it - how much would the tshirt be then?
That's exactly how you'd implement the above function, so probably not just with a simple addition :)